24H1KR3: Store architecture (#1171)
vtexdocs/dev-portal-content: docs/guides/VTEX-Platform-Overview/store-architecture/store-architecture-use-cases.md

---
title: "Store architecture use cases"
slug: "store-architecture-use-cases"
hidden: false
createdAt: "2024-05-23T13:08:55.338Z"
updatedAt: "2024-05-23T13:08:55.338Z"
excerpt: "Learn how our store architecture models are tailored to meet diverse business needs."
seeAlso:
 - "/docs/guides/understanding-vtex-reference-architectures"
 - "/docs/guides/store-architecture"
---

Store architecture communicates project vision, translating business needs into software requirements, and offering references for technical and business teams. This guide will explore the main VTEX reference architectures to learn how the VTEX platform integrates with different digital commerce strategies.

> >ℹ️ The diagrams were conceived according to the [Understanding VTEX reference architectures](https://developers.vtex.com/docs/guides/understanding-vtex-reference-architectures) guide.

Reviewer: ```suggestion
>ℹ️ Refer to [Understanding VTEX reference architectures](https://developers.vtex.com/docs/guides/understanding-vtex-reference-architectures) for more details about the structure of the diagrams presented in this guide.
```

## Business-to-customer (B2C)

> The business-to-customer (B2C) is the most common business model in VTEX and the base for the other business models.

Reviewer: ```suggestion
Business-to-customer (B2C) is the most common business model within VTEX customers. It serves as a foundation for other models.
```

> The diagram below shows the main components of a B2C commerce implementation, highlighting in pink the ones offered out-of-the-box (OOTB) by VTEX, also showing the other parts that may come from customers’ internal systems or third-party providers.

Reviewer: ```suggestion
The diagram below shows the main components of a B2C commerce implementation. Components highlighted in pink are offered out-of-the-box (OOTB) by VTEX, but the diagram also shows other parts that may come from customers' internal back office systems or third-party providers.
```

> ![image](https://cdn.jsdelivr.net/gh/vtexdocs/dev-portal-content@main/docs/guides/VTEX-Platform-Overview/store-architecture/basic-b2c.png)

Reviewer: I couldn't find this image in the folder, please add it

> >ℹ️ This diagram represents one single VTEX account and assumes that the customer is using the VTEX storefront solution.

Reviewer: Which VTEX storefront solution? Store Framework? Legacy Portal CMS? Fast Store? Any of those?

> Below is an integration flow diagram in a B2C architecture, where the asynchronous calls (not in real time) are represented by the black arrows, and synchronous calls represented by the blue arrows (real time).

Reviewer: ```suggestion
Below is an integration flow diagram in a B2C architecture, where the asynchronous calls (not in real time) are represented by black arrows, and synchronous calls are represented by blue arrows (real time).
```

![image](https://cdn.jsdelivr.net/gh/vtexdocs/dev-portal-content@main/docs/guides/VTEX-Platform-Overview/store-architecture/b2c-integration-flow.png)

An example of an asynchronous call is the stock update, which is represented by a black arrow pointing from the ERP to the Inventory module. A real-time update of rates in a payment is represented by a blue arrow, with information exchange in both blocks. This last scenario is synchronous because it is a sensitive stage of the customer’s purchase flow.

> The architecture suggested here may go further than just B2C, as you can see in the following diagrams.

Reviewer: ```suggestion
The architecture suggested here may go beyond B2C, as you can see in the following sections.
```

## Business-to-business (B2B) with B2B Suite

The main characteristic of a [business-to-business (B2B)](https://help.vtex.com/en/tutorial/b2b-overview--5vb9SNXhX2bZnkpAh7ADdC) model is that transactions occur between two or more legal entities.

The diagram below presents an example where the business model is a B2B using the [B2B Suite](https://developers.vtex.com/docs/apps/vtex.b2b-suite), which is represented within the **B2B Core Modules**. In this architecture, the main account acts as a marketplace for other VTEX sellers.

> ![image](https://cdn.jsdelivr.net/gh/vtexdocs/dev-portal-content@main/docs/guides/VTEX-Platform-Overview/store-architecture/basic-b2b.png)

Reviewer: Couldn't find this image in the folder, please add it

> The B2B Suite is a set of VTEX IO apps tailored to streamline B2B store management. This collection of apps empower the store to oversee organizations (the companies enable to make purchases in the store), as well as other functions and permissions for storefront and checkout. For instance, it prevents an organization from changing its address, offers special price tables, custom payments methods, user roles, among others settings.

Reviewer: ```suggestion
The B2B Suite is a set of VTEX IO apps tailored to streamline B2B store management. This collection of apps empower the store to oversee organizations (the companies enabled to make purchases in the store), as well as other functions and permissions for storefront and checkout. For instance, the store can offer specific price tables, collections, and custom payments methods for organizations, customers can have different user roles within organizations, among other features.
```

The main account has more modules than the sellers, sharing just some, such as the [Order Management System (OMS)](https://developers.vtex.com/docs/guides/orders-overview) and [Catalog](https://developers.vtex.com/docs/guides/catalog-overview), as you can see in the diagram below.

![image](https://cdn.jsdelivr.net/gh/vtexdocs/dev-portal-content@main/docs/guides/VTEX-Platform-Overview/store-architecture/basic-b2b-suite.png)

Declaring these modules in the architecture not only makes the process clearer but also makes the visualization of player responsibilities easier to understand. In this case, for example, [Master Data](https://developers.vtex.com/docs/guides/master-data-introduction) remains with the sellers because customer approval is done at the seller level.

In B2B scenarios, real-time communication between the B2B Suite and sellers is common, as the information is always up-to-date, and also for any calculations of fees and taxes at checkout, since the end customer needs to have a smooth purchasing experience.

> In the diagram above you can see the synchronous communication between [Organizations](https://developers.vtex.com/docs/apps/vtex.b2b-organizations) component in the main account and the sellers.

Reviewer: ```suggestion
In the diagram above you can see the synchronous communication between the [Organizations](https://developers.vtex.com/docs/apps/vtex.b2b-organizations) component in the main account and the sellers.
```

### Why choose this architecture?

> - **Streamlined Organization management**: Fosters efficient management of multiple organizations from a centralized platform by grouping storefront users into organizations and dividing them into cost centers. Shared order history, payment terms, and price lists enhance efficiency and promote cohesive collaboration across the organization.

Reviewer: Link to specific B2B documentation about this https://developers.vtex.com/docs/apps/vtex.b2b-organizations

- **Customized user experience**: Tailor the user experience for each organization or user group, providing personalized pricing, payment methods, and product catalogs to meet specific business needs.

> - **Enhanced security and control**: Assign roles and permissions to users, ensuring appropriate access levels and maintaining control over sensitive information and transactions.

Reviewer: Link to https://developers.vtex.com/docs/apps/vtex.storefront-permissions-ui

> - **Enhanced security and control**: Assign roles and permissions to users, ensuring appropriate access levels and maintaining control over sensitive information and transactions.

Reviewer: ```suggestion
- **Enhanced security and control**: Assign roles and permissions to B2B customers, ensuring appropriate access levels and maintaining control over sensitive information and transactions.
```

> - **Efficient Order Processing**: Simplify the checkout process with shared shipping addresses for cost centers, reducing eros and streamlining order fulfillment.

Reviewer: ```suggestion
- **Efficient Order Processing**: Simplify the checkout process with shared shipping addresses for cost centers, reducing errors and streamlining order fulfillment.
```

> - **Efficient Order Processing**: Simplify the checkout process with shared shipping addresses for cost centers, reducing eros and streamlining order fulfillment.

Reviewer: Maybe link to https://developers.vtex.com/docs/apps/vtex.b2b-checkout-settings

> - **Improved communication and collaboration**: Facilitate communication between users within an organization and streamline the creation of new organizations through request forms and approval queues.

Reviewer: "Facilitate communication between users within an organization" - how?

Maybe explain that they all get to see and manage the orders within their organization (or cost center?) provided that they have the required roles

Reviewer: In the part about creation of new organizations / request and approval queues, maybe link https://developers.vtex.com/docs/apps/vtex.b2b-organizations

### Learn more

- [B2B Suite - Overview](https://help.vtex.com/pt/tutorial/b2b-suite-overview--5eG6UfveWrai7looK0kVG3)

## Franchise accounts (Omnichannel)

This is a very common architecture model that integrates physical stores with ecommerce as [franchise accounts](https://help.vtex.com/en/tutorial/what-is-a-franchise-account--kWQC6RkFSCUFGgY5gSjdl) in VTEX.

A franchise account is linked to a main account and has the following characteristics, which you can see in the reference architecture below:

> - **No separate website**: Franchise accounts do not have their own dedicated website, as they operate within the main account’s one, effectively acting as part of a larger [marketplace](https://help.vtex.com/pt/tutorial/o-que-e-um-marketplace--680lLJTnmEAmekcC0MIea8).

Reviewer: ```suggestion
- **No separate website**: Franchise accounts do not have their own dedicated website, as they operate within the main account's ecommerce, effectively acting as part of a larger [marketplace](https://help.vtex.com/en/tutorial/what-is-a-marketplace--680lLJTnmEAmekcC0MIea8).
```

- **Customer Data Storage**: Customer data is stored in the main account’s [Master Data](https://developers.vtex.com/docs/guides/master-data-introduction).
- **Seller Type**: Each franchise account automatically functions as a [white label seller](https://help.vtex.com/en/tutorial/seller-white-label--5orlGHyDHGAYciQ64oEgKa) within the main account.
- **Catalog**: Franchise accounts inherit their [catalog](https://developers.vtex.com/docs/guides/catalog-overview) from the main account.

> - **Logistics and OMS**: Each franchise account has its own [Logistics](https://developers.vtex.com/docs/guides/fulfillment) and performs its own [Order Management](https://developers.vtex.com/docs/guides/orders-overview).

Reviewer: ```suggestion
- **Logistics and OMS**: Each franchise account has its own [Logistics](https://developers.vtex.com/docs/guides/fulfillment) settings and performs its own [Order Management](https://developers.vtex.com/docs/guides/orders-overview).
```

> - **Prices and Payments**: Each franchise can have its own [price](https://developers.vtex.com/docs/guides/pricing-overview) and [payment](https://developers.vtex.com/docs/guides/payments-overview) methods, or you can choose to inherit them from the main account.

Reviewer: ```suggestion
- **Prices and Payments**: Each franchise can have its own [price](https://developers.vtex.com/docs/guides/pricing-overview) and [payment](https://developers.vtex.com/docs/guides/payments-overview) methods, or they can inherit prices and payment methods from the main account.
```

> - **Promotions**: It is possible to create [promotions](https://developers.vtex.com/docs/guides/promotions-overview) for each franchise account and for the main one.

Reviewer: ```suggestion
- **Promotions**: It is possible to create [promotions](https://developers.vtex.com/docs/guides/promotions-overview) for each franchise account and for the main account.
```

> ![image](https://cdn.jsdelivr.net/gh/vtexdocs/dev-portal-content@main/docs/guides/VTEX-Platform-Overview/store-architecture/franchises-omni.png)

Reviewer: Please add the image

> This architecture model works well for brands that have multiple physical stores, franchisees, or representatives. In these cases, each of them can have a franchise account in VTEX integrated into the brand’s main account. This allows them to deliver the products sold through the brand’s ecommerce from an Omnichannel perspective.

Reviewer: ```suggestion
This architecture model works well for brands that have multiple physical stores, franchises, or representatives. In these cases, each physical store / franchise / representative can have a franchise account in VTEX integrated into the brand’s main account. This allows them to deliver the products sold through the brand’s ecommerce from an omnichannel perspective.
```

### Why choose this architecture?

- Potential to reduce the percentage of out-of-stock items.

> - Possible gain from inventory reduction through the use of the [Endless Aisle](https://help.vtex.com/en/tracks/estrategias-de-comercio-unificado--3WGDRRhc3vf1MJb9zGncnv/40KMlmGI5tN0r0KPCDWgGn?&utm_source=autocomplete) feature.

Reviewer: ```suggestion
- Possible gain from inventory reduction through the use of the [Endless Aisle](https://help.vtex.com/en/tracks/unified-commerce-strategies--3WGDRRhc3vf1MJb9zGncnv/40KMlmGI5tN0r0KPCDWgGn) feature.
```

> - Possible gain from inventory reduction through the use of the [Endless Aisle](https://help.vtex.com/en/tracks/estrategias-de-comercio-unificado--3WGDRRhc3vf1MJb9zGncnv/40KMlmGI5tN0r0KPCDWgGn?&utm_source=autocomplete) feature.

Reviewer: the Endless Aisle feature = Sales App, right? Is it available outside Brazil and Latin America? If not, I think it's worth adding the disclaimer to specify the regions where this is possible at the moment

- More shipping options, with the potential to reduce logistics costs.

> - The physical stores can be configured as [pick-up points](https://help.vtex.com/en/tracks/unified-commerce-strategies--3WGDRRhc3vf1MJb9zGncnv/4hXfgqXxS1lwAfnxgja3xW).

Reviewer: ```suggestion
- Physical stores can be configured as [pick-up points](https://help.vtex.com/en/tracks/unified-commerce-strategies--3WGDRRhc3vf1MJb9zGncnv/4hXfgqXxS1lwAfnxgja3xW).
```

> - The physical stores can be configured as [pick-up points](https://help.vtex.com/en/tracks/unified-commerce-strategies--3WGDRRhc3vf1MJb9zGncnv/4hXfgqXxS1lwAfnxgja3xW).

Reviewer: Same comment about the Sales App availability

> - The physical stores can function as small warehouses, operating in the [ship from store](https://help.vtex.com/en/tracks/unified-commerce-strategies--3WGDRRhc3vf1MJb9zGncnv/50GAmxxFsJoLWqcnMysWdl?&utm_source=autocomplete) strategy.

Reviewer: ```suggestion
- Physical stores can function as small warehouses, operating in the [ship from store](https://help.vtex.com/en/tracks/unified-commerce-strategies--3WGDRRhc3vf1MJb9zGncnv/50GAmxxFsJoLWqcnMysWdl) strategy.
```

> - The physical stores can function as small warehouses, operating in the [ship from store](https://help.vtex.com/en/tracks/unified-commerce-strategies--3WGDRRhc3vf1MJb9zGncnv/50GAmxxFsJoLWqcnMysWdl?&utm_source=autocomplete) strategy.

Reviewer: Same comment about the Sales App availability

### Learn more

> - [Unified Commerce Strategies](https://help.vtex.com/en/tracks/estrategias-de-comercio-unificado--3WGDRRhc3vf1MJb9zGncnv/2LGAiUnHES1enjHsfi8fI3)

Reviewer: ```suggestion
- [Unified Commerce Strategies](https://help.vtex.com/en/tracks/unified-commerce-strategies--3WGDRRhc3vf1MJb9zGncnv/2LGAiUnHES1enjHsfi8fI3)
```

## Multi-language and Multi-currency

> Some of our customers are present in multiple countries and want to have every local store running on VTEX. We provide a multi-language and multi-currency architecture to serve this customer, as you can see below.

Reviewer: ```suggestion
Some of our customers are present in multiple countries and want to have every local store running on VTEX. We provide a multi-language and multi-currency architecture for this scenario, explained in the following sections.
```

### Single account, multi-binding

In this architecture, a single VTEX account is configured to support multiple languages and currencies, using a [multidomain](https://help.vtex.com/en/tracks/vtex-store-overview--eSDNk26pdvemF3XKM0nK9/4yPqZQyj0t675QpcG7H6yl#multistore) approach.

The reference architecture below exemplifies its main characteristics:

> - **Separated website**: One single account, but each store with its own domain, bound to different [sales channels](https://help.vtex.com/en/tracks/vtex-store-overview--eSDNk26pdvemF3XKM0nK9/4EPwTXx5oFdSG1dA3zIchz#trade-policy) by [bindings](https://help.vtex.com/en/tutorial/what-is-binding--4NcN3NJd0IeYccgWCI8O2W?&utm_source=autocomplete).  

Reviewer: ```suggestion
- **Separated website**: One single account, but each store with its own domain, bound to different [trade policies](https://help.vtex.com/en/tracks/vtex-store-overview--eSDNk26pdvemF3XKM0nK9/4EPwTXx5oFdSG1dA3zIchz#trade-policy) by [bindings](https://help.vtex.com/en/tutorial/what-is-binding--4NcN3NJd0IeYccgWCI8O2W).  
```

Reviewer: _Trade policy_ is the current term for this, as defined in the [Localization glossary](https://docs.google.com/spreadsheets/d/1vSSg5CrimqSxvWjBoXLE5P7ATJCdRnPT/edit#gid=1978438735&range=A409).

> - **Search**: [Settings](https://developers.vtex.com/docs/guides/search-overview) are the same for all websites.

Reviewer: ```suggestion
- **Search**: [Search settings](https://developers.vtex.com/docs/guides/search-overview) are the same for all websites.
```

> - **CMS**: Shared access to [Content Management System (CMS)](https://developers.vtex.com/docs/guides/vtex-io-documentation-cms) between websites.

Reviewer: What if they don't use Store Framework?

- **Customer Data Storage**: One single [Master Data](https://developers.vtex.com/docs/guides/master-data-introduction) for all stores.

> - **Catalog**: One single catalog, segmented by sales channels.

Reviewer: ```suggestion
- **Catalog**: One single catalog, segmented by trade policies.
```

- **Logistics**: Each store has its logistics operation managed by different warehouses in the same Logistics panel.

> - **Order Management System (OMS)**: One single Logistics and OMS for all stores.

Reviewer: The Logistics part seems contradictory in these two topics

> - **Payments**: One Payments panel for all stores.

Reviewer: Maybe replace with "Shared Payment settings between all stores."?

> - **Prices and Promotions**: It is possible to configure different prices and promotions for each store, as you can segment by the sales channel. But prices and promotions for all stores are managed in the same panel.

Reviewer: ```suggestion
- **Prices and Promotions**: It is possible to configure different prices and promotions for each store, as you can segment them by trade policy. Still, prices and promotions for all stores are managed in the same panel.
```

> - **Message Center**: One single [Message Center](https://help.vtex.com/en/tutorial/conhecendo-o-message-center--tutorials_84) for all stores.

Reviewer: ```suggestion
- **Message Center**: One single [Message Center](https://help.vtex.com/en/tutorial/understanding-the-message-center--tutorials_84) for all stores.
```

> ![image](https://cdn.jsdelivr.net/gh/vtexdocs/dev-portal-content@main/docs/guides/VTEX-Platform-Overview/store-architecture/multi-currency-language-single-account.png)

Reviewer: Please add the image

#### Why choose this architecture?

- Supports [multi-currency](https://developers.vtex.com/docs/guides/vtex-io-documentation-internationalizing-product-prices) natively.

> - Supports multi-language with [Messages](https://developers.vtex.com/docs/guides/storefront-content-internationalization) and [Catalog Translation App](https://developers.vtex.com/docs/guides/catalog-internationalization).

Reviewer: Based on these topics, I think only stores on Store Framework can operate in this architecture, right? Maybe it's worth mentioning that in a callout in the beginning of this section

> >⚠ Product names are not translated on the [Checkout](https://help.vtex.com/en/tutorial/checkout-vtex-overview--7wcprkM7yZUflOqbzAN5SI) and [My Account pages](https://help.vtex.com/en/tutorial/how-my-account-works--2BQ3GiqhqGJTXsWVuio3Xh?&utm_source=autocomplete), as these modules call directly the catalog API to get product data, so the information does not pass through the Messages API.

Reviewer: ```suggestion
>⚠ Product names are not translated on the [Checkout](https://help.vtex.com/en/tutorial/checkout-vtex-overview--7wcprkM7yZUflOqbzAN5SI) and [My Account](https://help.vtex.com/en/tutorial/how-my-account-works--2BQ3GiqhqGJTXsWVuio3Xh) pages. These modules directly call the Catalog API to get product data, so the information does not pass through the Messages API.
```

This model is easier to set up, integrate, and maintain compared to the [multi-account](#multi-account) approach, but it may not be efficient for large and complex operations. This architecture is recommended for operations present in more than one country, requiring the configuration of different languages and currencies, but without the need for data segregation, as all core services are shared. Therefore, it may work well for operations managed by a single, centralized team.

### Multi-account

> In this architecture, there is a main VTEX account that will act as a [seller](https://help.vtex.com/en/tutorial/o-que-e-um-seller--5FkLvhZ3Few4CWWIuYOK2w) in all secondary VTEX accounts, acting as [marketplaces](https://help.vtex.com/en/tutorial/configurando-marketplace-entre-lojas-vtex). The main account - the seller - will be the source of truth for the [VTEX Core services](https://developers.vtex.com/docs/guides/getting-started#vtex-core-services), such as catalog, pricing, logistics, etc.

Reviewer: ```suggestion
In this architecture, one main account will act as a [seller](https://help.vtex.com/en/tutorial/what-is-a-seller--5FkLvhZ3Few4CWWIuYOK2w) in secondary accounts that act as [marketplaces](https://help.vtex.com/en/tutorial/configuring-the-marketplace-between-vtex-stores--tutorials_6520). The main account - the seller - will be the source of truth for the [VTEX Core services](https://developers.vtex.com/docs/guides/getting-started#vtex-core-services), such as catalog, pricing, logistics, and others.
```

The reference architecture below exemplifies its main characteristics:

- **Separated VTEX account**: Each store has its own VTEX account.

> - **Separated website**: Each store has its own website and may customize its frontend with a different look and feel per store.

Reviewer: ```suggestion
- **Separated website**: Each store has its own website and may customize its frontend with a different style per store.
```

> - **Customer Data Storage**: Segregated by marketplaces' account. There is no customer information within the main account’s [Master Data](https://developers.vtex.com/docs/guides/master-data-introduction).

Reviewer: ```suggestion
- **Customer Data Storage**: Segregated by marketplace account. There is no customer information within the main account’s [Master Data](https://developers.vtex.com/docs/guides/master-data-introduction).
```

> - **Catalog**: Separated catalog management, providing a unique assortment per store, defined based on the main account’s catalog.

Reviewer: "defined based on the main account’s catalog." - I didn't understand this part

Author: The main account is the owner of the catalog. Each store can define its assortment according to its needs, but always considering the catalog provided by the main. It is not possible to sell some product that is not in the main account's catalog.

- **Logistics and Order Management System (OMS)**: Independent Logistics and OMS per store.

> - **Fulfillment**: Supports single or multiple fulfillment.

Reviewer: What is single or multiple fulfillment?

Author: In this multi-account scenario, a single order can consist of items from one or multiple accounts. I will make this clearer in the document.

- **Payments**: Independent Payments management per store.
- **Pricing**: Independent Pricing management per store.

> - **Promotions**: Independent Promotions system per store.

Reviewer: ```suggestion
- **Promotions**: Independent Promotions management per store.
```

> - **Message Center**: Independent Message Center per store

Reviewer: ```suggestion
- **Message Center**: Independent Message Center per store.
```

> - **Message Center**: Independent Message Center per store

Reviewer: Maybe group all these topics into one

> ![image](https://cdn.jsdelivr.net/gh/vtexdocs/dev-portal-content@main/docs/guides/VTEX-Platform-Overview/store-architecture/multi-currency-language-multi-account.png)

Reviewer: Please add the image

#### Why choose this architecture?

> - The seller (main account) as catalog owner and marketplaces per country allows a way to make a multi-language solution, as each store can manage the language and currency locally, in its own store.

Reviewer: ```suggestion
- Having the seller (main account) act as catalog owner and having marketplaces per country allows a way to make a multi-language solution, as each store can manage the language and currency locally, in its own store.
```

> - Each account maintains its own catalog information since the information coming from the main account is always in the standard language set there.

Reviewer: Not sure I understand this

> - Support better translations compared to the [single account, multi-binding](#single-account-multi-binding) model, as there is an **independent catalog translation**. Therefore, you do not need to rely on Messages or Catalog translation app.

Reviewer: Link to docs about Messages and Catalog translation app

> >⚠ [Assembly options](https://developers.vtex.com/docs/guides/assembly-options-app), [attachments](https://developers.vtex.com/docs/guides/cart-attachments-section-api-quick-start-guides), and [services](https://developers.vtex.com/docs/guides/vtex-io-documentation-service) are not supported.

Reviewer: I think "attachments" here actually means the Catalog attachments feature (related to assembly options, basically a free version of services). So I would recommend linking a doc like [this](https://help.vtex.com/tutorial/what-is-an-attachment--aGICk0RVbqKg6GYmQcWUm) instead of that one about cart attachments in Checkout

> Promotions from the main account have limited capabilities, as every information that comes from the customer will not be accessible by the main account. For example, if you want to create promotions for a customer cluster in the main account, it will not work because it does not have access to the marketplace’s Master Data.

Reviewer: I would add this information elsewhere, not within "Why choose this architecture?"

This architecture is recommended for operations in more than one country, requiring the configuration of different languages and currencies, where different teams manage each localized store.

### Learn more

- [Accounts and architecture](https://help.vtex.com/en/tracks/vtex-store-overview--eSDNk26pdvemF3XKM0nK9/4yPqZQyj0t675QpcG7H6yl)

> - [Choosing between a multi-store architecture or an additional environment](https://help.vtex.com/en/tutorial/escolhendo-entre-arquitetura-multi-loja-ou-ambiente-adicional--4HRNpa1OCKZ5YzP8yiilBL)

Reviewer: ```suggestion
- [Choosing between a multi-store architecture or an additional environment](https://help.vtex.com/en/tutorial/choosing-between-a-multi-store-architecture-or-an-additional-environment--4HRNpa1OCKZ5YzP8yiilBL)
```
